graphitepager/graphite_data_record.py

```diff
@@ -8,7 +8,12 @@ class NoDataError(ValueError):
 class GraphiteDataRecord(object):
 
     def __init__(self, metric_string):
-        meta, data = metric_string.split('|')
+        meta, data = None, None
+        try:
+            meta, data = metric_string.rsplit('|',1)
```

Comment:
The `m,d = str.rsplit('|',1)` won't fail when there are many `|` in the string, and it will split on the last such `|` found.